subjects/fishandchips/README.md

```diff
@@ -0,0 +1,40 @@
+## Fishandchips
```

Comment:
The title should be in lower case `fishandchips`

Reply:
it's fixed